lntypes/hash.go

```diff
@@ -19,30 +19,30 @@ func (hash Hash) String() string {
 
 // NewHash returns a new Hash from a byte slice.  An error is returned if
 // the number of bytes passed in is not HashSize.
-func NewHash(newHash []byte) (*Hash, error) {
+func NewHash(newHash []byte) (Hash, error) {
```

Comment:
In a few other areas within the codebase, we use a `Make` prefix to indicate that we're returning a concrete type, and a `New` to indicate that a pointer will be returned. Not a blocker here to switch over, but something that we may want to consider standardizing throughout the codebase.

Reply:
Renamed to `Make`